google-account-plugin/src/com/google/cloud/tools/intellij/stats/LoginTracking.java

```diff
@@ -27,4 +27,5 @@ private LoginTracking() {
   public static final String LOGIN_START = "user.login.start";
   public static final String LOGIN_CANCELLED = "user.login.cancelled";
   public static final String LOGIN_COMPLETE = "user.login.complete";
```

Comment:
Yes, in the same file, [here](https://github.com/GoogleCloudPlatform/gcloud-intellij/blob/master/google-account-plugin/src/com/google/cloud/tools/intellij/login/IntellijGoogleLoginService.java#L481).